### PR TITLE
Encode the command string as a Windows wide-character string (UTF-16LE)

### PR DESCRIPTION
**Pull request by rgl**
This addresses https://github.com/masterzen/winrm/issues/96

### CODE REVIEW

File: powershell.go [hunk 1]
```diff
@@ -3,21 +3,28 @@ package winrm
...
 
 	// Create the powershell.exe command line to execute the script
 	return fmt.Sprintf("powershell.exe -EncodedCommand %s", encodedCmd)
 }
+
+func encodeUtf16Le(s string) []byte {
+	d := utf16.Encode([]rune(s))
+	b := make([]byte, len(d)*2)
+	for i, r := range d {
+		b[i*2] = byte(r)
```
Comment (masterzen) [inline powershell.go]: I think to be on the safe side that it would be better to use:
```
binary.LittleEndian.PutUint16(b[i*2:], r)
```

Comment (rgl) [reply powershell.go]: Why do you think its unsafe?

Comment (masterzen) [reply powershell.go]: Maybe safe wasn't the correct word. It's definitely easier to use the standard library way of doing things and possibly correct on all platforms.

**Comment by masterzen** [comment]
I'm closing this PR in favor of #111. Thanks for your help on this topic.

**Pull request closed without merging**